packages/atlas/src/components/_inputs/ComboBox/ComboBox.tsx

```diff
@@ -70,15 +70,10 @@ export const ComboBox = <T extends unknown>(props: ComboBoxProps<T>) => {
       }
     },
     onInputValueChange: ({ inputValue }) => {
-      if (!inputValue) {
```

Comment:
After changes to this file, whitelisting in NFT sale form doesn't work. No requests are fired after I type in member name